src/style.css

```diff
@@ -1,3 +1,72 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+
+  --main-color: #00ACDC;
+}
+
+.header__content {
+  display: flex;
+  margin: 0;
+  justify-content:space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.header__logoBox {
+  height: 40px;
+  width: 40px;
+}
+
+.header__nav{
+  margin: 0;
+}
```

Comment:
there is no need to set `margin: 0`, works without this style as expected (not critical)